themes/bootstrap3/templates/record/view.phtml

```diff
@@ -32,7 +32,7 @@
   <?php $sidebarList = $this->related()->getList($this->driver); ?>
   <div class="<?=$this->layoutClass('mainbody')?><?=count($sidebarList) < 1 ? ' solo' : '' ?>">
     <input type="hidden" value="<?=$this->escapeHtmlAttr($this->driver->getUniqueId())?>" class="hiddenId" />
-    <input type="hidden" value="<?=$this->escapeHtmlAttr($this->driver->getSourceIdentifier()) ?>" class="hiddenSource" />
+    <input type="hidden" value="<?=$this->escapeHtmlAttr($this->driver->getSourceIdentifier())?>" class="hiddenSource" />
```

Comment:
If no space is the style, I apologize for recent inconsistencies!

Reply:
I don't think we have a formal style rule, but I think "no space" is more common. In any case, I made this change simply to make the two view.phtml files consistent with one another, and the space seemed to be the less-used variation, so I removed it. It would be nice if php-cs-fixer could enforce this, because otherwise it's impossible to remember to make it perfectly uniform all the time. I haven't checked to see if there's a rule that applies (no time at the moment) but it might be worth doing.